manifests/globals.pp

```diff
@@ -2,10 +2,26 @@
 #
 # @param scl
 #   Use a specific Software CoLlection on Red Hat based systems
+#
+# @param version
+#   The version of Redis to install and manage.
+#
+# @param manage_dnf_module
+#   Manage the DNF module. This only makes sense on distributions that use DNF
+#   package manager, such as EL8 or Fedora. It also requires Puppet 5.5.20+ or
+#   Puppet 6.15.0+ since they ship the dnfmodule provider.
 class redis::globals (
   Optional[String] $scl = undef,
+  $version = '6',
```

Comment:
It may be confusing because the user can also pass in a version via `package_ensure`. There is also the `redis_server_version` fact (https://github.com/voxpupuli/puppet-redis/blob/master/lib/facter/redis_server_version.rb).

Should this be more focused and instead be `Optional[String[1]] $dnf_module_stream = undef` (and drop the `manage_dnf_module` parameter)?